OWNERS

```diff
@@ -0,0 +1,8 @@
+maintainers:
```

Comment:
What's the correct syntax for the OWNERS file? In https://github.com/kubernetes/kubernetes/blob/master/OWNERS there are approvers, and in https://github.com/kubernetes/helm/blob/master/OWNERS there are maintainers. In other repos (e.g. kops), they only have assignees.